packages/sit-onyx/src/index.ts

```diff
@@ -51,6 +51,8 @@ export { default as OnyxSkeleton } from "./components/OnyxSkeleton/OnyxSkeleton.
 export { default as OnyxSwitch } from "./components/OnyxSwitch/OnyxSwitch.vue";
 export * from "./components/OnyxSwitch/types";
 
+export { default as OnyxTable } from "./components/OnyxTable/OnyxTable.vue";
```

Comment:
Can you also create a types.ts file with the `OnyxTableProps`?

Reply:
we have non yet, should it be an empty object?

Reply:
not sure if I should already setup e.g. the dencity props